Solutions/Recorded Future/Playbooks/RecordedFuture-URL-Recently_Reported_by_Insikt_Group-IndicatorProcessor.json

```diff
@@ -0,0 +1,192 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+		"comment": "This playbook leverages the Recorded Future API and automatically imports the Recently Reported by Insikt Group URL RiskList, as Threat Intelligence Indicators, for detection purposes in Azure Sentinel",
+		"author": "Adrian Porcescu, Recorded Future"
+	},
+	"parameters": {
+        "PlaybookName": {
+            "defaultValue": "RecordedFuture-URL-Recently_Reported_by_Insikt_Group-IndicatorProcessor",
+            "type": "String"
+        },
+        "PlaybookNameBatching": {
+            "defaultValue": "RecordedFuture-ImportToSentinel",
+            "type": "String"
+        }
+    },
+    "variables": {
+        "RecordedFutureConnectionName": "[concat('recordedfuture-', parameters('PlaybookName'))]"
```

Comment:
Please reduce the playbook name. As after concatenation it's become more than 80 chars which is not acceptable during deployment.